### PR TITLE
Initial implementation volume API

### PR DESCRIPTION
**Pull request by stanphong**
Add new volume function in speaker.js in api function folder and speaker.js in Speaker folder
Volume bar works in front-end and sends the value back to the API
API does not actually work yet.

image

### CODE REVIEW

**Comment by stanphong** [comment]
Update front-end with forward, reverse control
image

File: src/Pages/Speaker/Speaker.js [hunk 1]
```diff
@@ -34,24 +38,57 @@ function SpeakersPage(props) {
...
+    } else {
+      await modifySpeakerWrapper(resume);
+    }
+    setIsPlaying(!isPlaying);
+  };
+
+  const handleVolumeChange = async (volume) => {
+    setVolumeState(volume); // Update local volume state
+    const result = await setVolume(volume, props.user.token);
+  };
```
Comment (evanugarte) [inline src/Pages/Speaker/Speaker.js]: will this get called any time the slider changes value, or when the user is done dragging the slider?

Comment (akshtshrma24) [reply src/Pages/Speaker/Speaker.js]: When user is done dragging slider

**Review by akshtshrma24 — APPROVED** [review]

File: src/Pages/Speaker/Speaker.js (hunk 1 above)
Comment (akshtshrma24) [reply src/Pages/Speaker/Speaker.js]: now it slike theat

**Review by akshtshrma24 — APPROVED** [review]

**Review by evanugarte — APPROVED** [review]

**Pull request merged**